assets/javascript/index.js

```diff
@@ -0,0 +1,215 @@
+/**
+ * URL of the manifest file.
+ */
+let _manifestURL;
+
+/**
+ * Data of the manifest file.
+ */
+let _manifestData;
+
+/**
+ * URL parameters.
+ */
+let _params;
+
+/**
+ * Root directory of the glTF assets relative to the manifest file.
+ */
+let _rootdir;
+
+const ENGINESURL = "assets/json/engines.json";
+
+/**
+ * Data of the rendering engines parsed from the engines json file.
+ */
+let _engineData;
+
+/** 
+ * Parse parameters from url
+ * @returns - parse parameters
+*/
+function parseParameters() {
+    let result = {};
+    let parameters = location.href.split("?")[1];
```

Comment:
Are we going to use `document.referrer` for this?

Reply:
That can work.  I'll tweak it to support that.

Reply:
I added in referrer support if `manifest` is not provided in the url